app/src/main/java/org/mozilla/fenix/settings/TrackingProtectionFragment.kt

```diff
@@ -31,7 +31,15 @@ class TrackingProtectionFragment : PreferenceFragmentCompat() {
         val preferenceTP = findPreference<Preference>(trackingProtectionKey)
         preferenceTP?.onPreferenceChangeListener =
             Preference.OnPreferenceChangeListener { _, newValue ->
-                requireComponents.core.updateTrackingProtection(newValue as Boolean)
+                val policy =
```

Comment:
nit: This seems to belong inside the with{} block.

Reply:
True! Thanks!